### PR TITLE
working chat feedback dump script (with api addition)

### PR DESCRIPTION
**Pull request by rkuo-danswer**
## Description
Fixes DAN-861.

## How Has This Been Tested?
[Describe the tests you ran to verify your changes]

## Accepted Risk
[Any know risks or failure modes to point out to reviewers]

## Related Issue(s)
[If applicable, link to the issue(s) this PR addresses]

## Checklist:
- [ ] All of the automated tests pass
- [ ] All PR comments are addressed and marked resolved
- [ ] If there are migrations, they have been rebased to latest main
- [ ] If there are new dependencies, they are added to the requirements
- [ ] If there are new environment variables, they are added to all of the deployment methods
- [ ] If there are new APIs that don't require auth, they are added to PUBLIC_ENDPOINT_SPECS
- [ ] Docker images build and basic functionalities work
- [ ] Author has done a final read through of the PR right before merge

### CODE REVIEW

File: backend/ee/danswer/server/query_history/api.py [hunk 1]
```diff
@@ -330,6 +333,36 @@ def snapshot_from_chat_session(
     )
 
 
+@router.get("/admin/chat-sessions")
```
Comment (pablonyx) [inline backend/ee/danswer/server/query_history/api.py:336]: nit: seems worth defining a new admin_router for the admin endpoints in query history api

File: backend/scripts/chat_feedback_dump.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
+# This file is used to demonstrate how to use the backend APIs directly
+# In this case, the equivalent of asking a question in Danswer Chat in a new chat session
```
Comment (pablonyx) [inline backend/scripts/chat_feedback_dump.py]: Is this accurate? Might be worth adding context about the feedback, assistants use case, etc?

File: backend/scripts/chat_feedback_dump.py [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+                logger.info(
+                    f"user={user_id} "
+                    f"session={session.id} "
+                    f"message={m.message} "
+                    f"feedback_type={m.feedback_type} "
+                    f"feedback_text={m.feedback_text}"
+                )
+
+
+if __name__ == "__main__":
```
Comment (pablonyx) [inline backend/scripts/chat_feedback_dump.py:221]: Might be worth adding a tiny comment to poke end user of the script in the right direction (something like "would need to retrieve the fastapiusersauth access token" and set that in the headers)

**Review by pablodanswer — APPROVED** [review]

File: backend/scripts/chat_feedback_dump.py (hunk 2 above)
Comment (rkuo-danswer) [reply backend/scripts/chat_feedback_dump.py]: oops, that was a leftover comment from the script i started with.

File: backend/ee/danswer/server/query_history/api.py (hunk 1 above)
Comment (rkuo-danswer) [reply backend/ee/danswer/server/query_history/api.py:336]: will leave this for later, out of scope for this.

**Pull request merged**